index.html

```diff
@@ -0,0 +1,82 @@
+    <title>Portfolio</title>
+</head>
+<body>
+  <section class="intro">
+
+<div class="logo">
+    <h1>Eros</h1>
+</div>
+  <div class="navbar">
+   <ul>
+       
+        <li><a href="Home">Home</a></li>
+        <li><a href="Porfolio">Portfolio</a></li>
+        <li><a href="Contact">Contact</a></li>
+
+    </ul>
+  </div>
+</section>
+<section >
+  <div class="description">
+    <img class="profile" src="./Images/Profile Photo.jpeg" alt="profile">
+    <h2 class="work"> I am Aime <br> a full stack software developer</h2>
+    <!-- <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur amet porro id, consectetur cumque enim earum illum officiis,</p> -->
+    <input class="btn" type="button" value="My Services">
+  </div>
+</section>
+<section >
+  <h3 class="services-intro">Get creative and enjoy our web and software services from us</h3>
+  <div class="services"> 
+  <div class="service-1">
+    <h3>Web Design</h3>
+    <p class="content">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Eos, commodi nulla obcaecati beatae cumque esse quos repudiandae reiciendis dicta omnis? Minima voluptates assumenda sequi deleniti adipisci magni ab porro quos.</p>
+</div>
+<div class="service-2">
+  <h3>Web development</h3>
+  <p class="content">Lorem ipsum dolor sit amet consectetur adipisicing elit. Alias assumenda reprehenderit voluptas perspiciatis officia repellendus provident odio non impedit aperiam? Ratione reiciendis quos sit hic ullam, ex cumque omnis nulla?</p>
+</div>
+<div class="service-3">
+  <h3>Software development</h3>
+  <p class="content">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ex incidunt optio, eos fuga, a beatae nisi quas vitae rem quasi inventore, eligendi cumque quo dolores! Libero, fuga iusto? Aperiam, dignissimos?</p>
+
+</div>
+</div>
+</section>
+<section class="feedback">
+  <h2>Feedback</h2>
+  <form action="feedback">
+    <label for="feeback">
+      <input class="name" type="text" name="name" value="name" placeholder="Enter your name" required><br><br>
+      <input class="email" type="email" name="email" value="email" placeholder="enter your email" required><br><br>
+      <textarea class="suggestion" name="feedback" id="feedback" placeholder= "feedback" cols="30" rows="10"></textarea><br>
+      <input class="submit" value="submit" value="submit">
+    </label>
+  </form>
+</section>
+<section>
+  <div class="contact">
+    <h2>Reach out for help:</h2>
+    <!-- <i class="material-icons">call</i> -->
+    <a href="#" class="fa fa-facebook"></a>
+    <a href="https://twitter.com/AimeSebagabo" class="fa fa-twitter"></a>
+    <a href="https://www.linkedin.com/in/jean-aime-sebagabo-279aa2207/" class="fa fa-linkedin"></a>
+    <a href="#" class="fa fa-instagram"></a>
+    <a href="https://github.com/Hazard10-eng" class="fa fa-github"></a>
+
+  </div>
+<section class="ending">
+  <footer>
+    <h1 class="end">MY PORTFOLIO</h1>
+  </footer>
+</section>
```

Comment:
- [ ] From the project requirements you should only build two sections i.e the header section and the headline section. So please kindly remove all these sections because I guess you suppose to build these sections in your next project.